l10n_nl_tax_statement/tests/test_l10n_nl_vat_statement.py

```diff
@@ -567,3 +567,15 @@ def test_20_multicompany(self):
 
         company_ids_full_list = statement_parent._get_company_ids_full_list()
         self.assertEqual(len(company_ids_full_list), 3)
+
+    def test_21_action_xls(self):
+        """Generate XLS report from action"""
+        report = "l10n_nl_tax_statement.action_report_tax_statement_xls_export"
+        self.report_action = self.env.ref(report)
+        self.assertEqual(self.report_action.report_type, "xlsx")
+        model = self.env["report.%s" % self.report_action["report_name"]].with_context(
+            active_model="l10n.nl.vat.statement"
+        )
+        res = model.create_xlsx_report(self.statement_1.ids, data=None)
+        self.assertTrue(res[0])
+        self.assertEqual(res[1], "xlsx")
```

Comment:
This test now does not result in a lot of coverage, because the report is quite empty.

Is it an idea to modify each existing test to run an XLS export in the end? That way, a lot more cases should be covered.

Reply:
I think is a good idea :+1: Thanks!

Reply:
Coverage of `l10n_nl_tax_statement_xlsx.py` increased from 66.15% to 100% :muscle: